app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,108 @@
+class ProjectsController < ApplicationController
+  def index
+    @projects = Project.all
+  end
+
+  def new
+    @project = Project.new
+  end
+
+  def create
+    compnay = Company.first
+    @project = Project.new
```

Comment:
Why do we need to initialize a new resource? Can we not use the one we initialized in the new action?

Reply:
NoMethodError (undefined method `name=' for nil:NilClass) if i dont initilaiza new recource